docker/puppeteer/lighthouserc.js

```diff
@@ -1,23 +1,24 @@
 module.exports = {
-    ci: {
-      collect: {
-        url: ['http://firm-deputy-hub:8888/supervision/deputies/firm/'],
-        settings: {
-          extraHeaders: JSON.stringify({Cookie: 'XSRF-TOKEN=abcde; Other=other'}),
-          chromeFlags: "--disable-gpu --no-sandbox",
-        },
+  ci: {
+    collect: {
+      url: ["http://firm-deputy-hub:8888/supervision/deputies/firm/"],
```

Comment:
Does this need to change to 8887 too?

Reply:
I don't _think_ so, I didn't want to change the URL when it was running through the CI pipeline which is where puppeteer is as I thought it might cause weird problems. I've only changed the locally running cypress URL not the puppeteer one.